ci/circle-36-locale.yaml

```diff
@@ -5,6 +5,7 @@ channels:
 dependencies:
   - beautifulsoup4
   - cython
+  - gcsfs
```

Comment:
let's remove from here to make sure the tests are skipped properly (as tested in 2.7 / 3.6 builds already)